spec/presenters/person_presenter_spec.rb

```diff
@@ -142,4 +143,83 @@
       expect(presenter.hovercard[:contact]).to have_key(:aspect_memberships)
     end
   end
+
+  describe "#profile_hash_as_api_json" do
+    let(:current_user) { FactoryGirl.create(:user,
+      profile: FactoryGirl.create(:profile_with_image_url)) }
```

Comment:
- Layout/AlignParameters: Align the parameters of a method call if they span more than one line.
- Layout/BlockEndNewline: Expression at 149, 61 should be on its own line.